examples/llama_index_examples/pyproject.toml

```diff
@@ -10,6 +10,7 @@ readme = "README.md"
 package-mode = false
 requires-python = ">=3.10,<3.13"
 dependencies = [
+    "pangea-multipass",
```

Comment:
`pangea-multipass-llama-index` should already include this dependency, it's not happening?

Reply:
I think I misinterpreted the error message I was getting:

```
Because pangea-multipass-llama-index (0.2.0) @ file:///Users/.../pangea-multipass/packages/pangea-multipass-llama-index depends on pangea-multipass (>=0.2.0) which doesn't match any versions, pangea-multipass-llama-index is forbidden.
So, because pangea-multipass-llama-index (0.2.0) @ file:///Users/.../pangea-multipass/packages/pangea-multipass-llama-index depends on llama-index-readers-jira (^0.3.1)
 and llama-index-examples depends on llama-index-readers-jira (0.4.1), version solving failed.
```

Updating the lock file appears to fix this - let me know if this looks right.